index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Awesome Books</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <header class="page-header">
+        <nav class="nav-bar">
+          <p>
+            <a href="#">Awesome Books</a>
+          </p>
+          <ul class="nav-menu">
+            <li class="nav-item">
+              <a href="#" class="nav-link" id="list-link">List</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link" id="add-link">Add New</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link" id="contact-link">Contact</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <div id="time">
+        <div class="time"></div>
+      </div>
+      <section class="books-section page-sections">
+        <h1 class="header">Awesome Books</h1>
+        <div class="books-container"></div>
+      </section>
+      <section class="add-book-section page-sections">
+        <h2 class="header">Add a new book</h2>
+        <hr />
+        <form class="books-form">
+          <input
+            class="form-input"
+            id="title"
+            type="text"
+            placeholder="Title"
+            required
+          />
+          <input
+            class="form-input"
+            id="author"
+            name="author"
+            type="text"
+            placeholder="Author"
+            required
+          />
+          <div class="add-btn-div">
+            <button id="add-btn" type="button">Add</button>
+          </div>
+        </form>
+      </section>
+      <section class="contact-section page-sections">
+        <h2 class="header">Contact Information</h2>
+        <hr />
+        <div class="contact-section-wrapper">
+          <p>
+            Do you have any questions or whant to say "Hello"?
```

Comment:
- [OPTIONAL] Ups! Typo in this sentence, "whant" 😅

Reply:
@GiuseppeTG Thanks for the insight, I will take care of it